try-it-out/docs/pages/choose-jenkins-pipeline-type.rst

```diff
@@ -0,0 +1,32 @@
+.. _Choose Jenkins Pipeline Type:
+
+------------------------------
+Choose a Jenkins Pipeline Type
+------------------------------
+
+Now that you have all the services you need running on Docker, you will now be choosing the type of Jenkins pipeline that you will be configuring for a Spring Boot API sample application that we created just for this guide.
+
+To begin, navigate to the homepage of your Jenkins instance.
+
+Click **New Item** on the left-hand menu to create a new Jenkins pipeline. 
+
+.. image:: ../images/choose-jenkins-pipeline-type/click_new_item.png
+
+For the **Item Name**, enter **sample-spring-boot-api**. Note that the **item name** you give it is arbitrary, but for consistency and referencing purposes, we'll be calling it **sample-spring-boot-api**. Below, you should see several options for the type of job to use. For the options that can currently be used with the Jenkins Templating Engine, they can be described as follows:
```

Comment:
I like having these docs somewhere, but i'm not positive this is the right place.  it's talking about concepts that haven't been introduced yet (pipeline configurations, etc)